update authors to blog post

maneesha opened the pull request:
Addresses this comment (https://github.com/carpentries/carpentries.org/pull/426#issuecomment-2684326468) by adding authors to the blog post.

kariljordan commented:
Thank you @anelda! I will be merging this post on Tuesday.

anelda commented:
Thanks @Dr. Jordan <kariljordan@me.com>

On Mon, Apr 8, 2019 at 9:54 PM Kari L. Jordan, PhD <notifications@github.com>
wrote:

> Thank you @anelda <https://github.com/anelda>! I will be merging this
> post on Tuesday.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/carpentries/carpentries.org/pull/428#issuecomment-480982221>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG4Xpt6gEmWE5y7PbaLXVt8YPtH-vIdTks5ve55vgaJpZM4ccneH>
> .
>

maneesha committed:
update authors to blog post

OscarSiba committed:
Update changes-to-self-organised-workshops.md

@maneesha I am so sorry about yet another last minute change, but something else we didn't pick up is that the subheadings were bolded * * instead of marked in subheading case ##. So I have changed that on the file, but please check to see it didn't affect your technical checks/merge.

OscarSiba reviewed (commented):
I am happy with these changes, thank you @maneesha.
